book/website/afterword/glossary.md

```diff
@@ -123,6 +123,13 @@ Contributors
 Corresponding author
   The person who administers an academic article for the research group. They are responsible for receiving the reviewers comments, the proofs, corresponding with the editors and their details are printed on the final version of the published article.
 
+Creative Commons
+  Creative commons licenses provide standard language with which to place a creative work into the commons by providing simple terms under which others can re-use it.
+  This is necessary as in almost all legal regimes by default authors reserve all rights to works even if they make them publicly available unless they explicitly provide a license stipulating otherwise.
+  For more on this refer to the chapter: {ref}`Licensing<rr-licensing>`.
+  There are a number of variants of the license which add terms to the CC0 or public domain license such as requiring attribution (BY), prohibiting commercial re-use (NC), prohibiting derivative works (ND), and requiring that any derivative works be shared under the same terms (SA).
```

Comment:
There have been some revisions to it which I think includes an internationalized variant intended to apply properly in more jurisdictions but I don't know if there are variants beyond these revisions over time.

Reply:
I think it's easiest to understand them as variants particularly because of the way they're abbreviated and the fact that they're all drafted by the Creative Commons (at least that's how I understand them anyway!)

Reply:
They are technically different licenses but the specific clauses with 2 letter abbreviation are effectively modular. I'll rephrase this a little though as the cc0 license is not the common core of these license it's a public domain dedication rather than a license per se. all the other variant have the same 'core' with some or modified different clauses.

Reply:
addressed in https://github.com/the-turing-way/the-turing-way/pull/3889/commits/f3ed3a5ab5ae68073c0019f253d5c32265ee023e